include/swift/AST/DiagnosticsParse.def

```diff
@@ -2009,6 +2009,9 @@ ERROR(macro_expansion_expr_expected_macro_identifier,PointsToFirstBadToken,
 ERROR(macro_expansion_decl_expected_macro_identifier,PointsToFirstBadToken,
       "expected a macro identifier for a pound literal declaration", ())
 
+ERROR(declaration_attr_expected_kind,PointsToFirstBadToken,
+      "expected a declaration macro kind ('.freestanding' or '.attached')", ())
```

Comment:
We should drop the leading "."'s here? (not a big deal)

Reply:
Stale error message. Fixed